reference-architecture/aws-ansible/ose-on-aws.py

```diff
@@ -72,7 +72,7 @@
 @click.option('--s3-bucket-name', help='Bucket name for S3 for registry')
 @click.option('--github-client-id', help='GitHub OAuth ClientID')
 @click.option('--github-client-secret', help='GitHub OAuth Client Secret')
-@click.option('--github-organization', prompt=True, multiple=True, help='GitHub Organization')
+@click.option('--github-organization', prompt=True, help='GitHub Organization')
```

Comment:
I was having difficulties with the variable getting trashed. I was either getting trashed like i posted in the comment above or '[u'openshift', u'RHsyseng']'


Still working through having more than 1 organization. As the space between organizations tends to cause the wrong value to be added and modifying the split makes the list print as ['openshift,RHSyseng']